--- gambatte_src-0.5.0-wip2v2/gambatte_qt/src/src.pro~	2011-10-30 21:59:06.000000000 +0100
+++ gambatte_src-0.5.0-wip2v2/gambatte_qt/src/src.pro	2012-03-09 07:53:09.274100677 +0100
@@ -40,4 +40,4 @@ TARGET = gambatte_qt
 macx:TARGET = "Gambatte Qt"
 DESTDIR = ../bin
 INCLUDEPATH += ../../libgambatte/include
-LIBS += -L../../libgambatte -lgambatte -lz
+LIBS += -L../../libgambatte -lgambatte -lz -ldl
